lib/transport/selenium-webdriver/httpclient.js

```diff
@@ -52,6 +62,7 @@ module.exports = function(settings, HttpResponse) {
       this.options.data = data;
       this.options.path = path;
       this.options.method = method;
+      this.options.redact = this.isDataRedacted(data);
```

Comment:
We should move this to `request.js` file as this method is solely for use in Selenium and so the redaction logic doesn't belong in this file.

Reply:
Updated in [0f6b9a7](https://github.com/nightwatchjs/nightwatch/pull/4066/commits/0f6b9a7e16ada3064d38a555bab1b6e85dffbe84)